connect/api/src/main/java/org/apache/kafka/connect/source/SourceTask.java

```diff
@@ -105,7 +105,7 @@ public void commit() throws InterruptedException {
      *
      * @param record {@link SourceRecord} that was successfully sent via the producer or filtered by a transformation
      * @throws InterruptedException
-     * @see commitRecord(SourceRecord, RecordMetadata)
+     * @see #commitRecord(SourceRecord, RecordMetadata)
```

Comment:
This has been fixed by: https://github.com/apache/kafka/pull/8291 
Please omit and rebase to get the latest changes. 

But see below on line 126 for a misformatting that needs to be fixed:
`@{link commitRecord(SourceRecord)}` needs to be `{@link #commitRecord(SourceRecord)}`